packages/sources/react/src/components/VtmnIcon/VtmnIcon.tsx

```diff
@@ -48,19 +48,32 @@ export const VtmnIcon: React.FC<VtmnIconProps> = ({
         return 'content-information';
       case 'warning':
         return 'content-warning';
-      case 'danger':
-        return 'content-danger';
+      case 'negative':
+        return 'content-negative';
+      case 'neutral':
+        return '';
     }
   };
 
+  const getIconColor = () => {
+    let iconColor = 'inherit';
+    if (color) {
+      iconColor = `var(--vtmn-color_${color})`;
+    } else if (variant !== 'neutral') {
+      iconColor = `var(--vtmn-semantic-color_${retrieveSemanticColor(
```

Comment:
I understand your proposal with "neutral" to avoid `inherit` by default. But, when `variant` will be `neutral`, this code will output `var(--vtmn-semantic-color_)`, we need to find a better solution, please.

Reply:
Hello @crotru, do you have an idea about this?
Thx!

Reply:
This actually works pretty fine.
The idea was to keep it black by default, but stating the color as inherit when a "neutral" variant is specified.

But...

Coming back to it, it actually makes more sense to keep the color as inherit (or not stated at all) when no variant is provided. 
I'll come up with something that way.

Reply:
Thanks a lot @crotru